libass/ass_fontselect.c

```diff
@@ -851,15 +910,21 @@ static char *select_font(ASS_FontSelector *priv,
         // TODO: consider changing the API to make more efficient
         // implementations possible.
         for (int i = 0; i < meta.n_fullname; i++) {
-            default_provider->funcs.match_fonts(default_provider->priv,
+            matched_font = default_provider->funcs.match_fonts(default_provider->priv,
```

Comment:
Currently, this leaks because I don't call ``ass_font_provider_free_fontinfo`` + ``free`` on the returned font.

If I free it, the result from ``get_font_result`` will point to invalid data.

I'm working on a fix.

Reply:
Fixed by b7cc7ccc9f2894e052d2758f6195bd5f5cac124c